vector/src/main/res/layout/view_rich_text_menu_button.xml

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ImageButton xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="48dp"
     android:layout_height="48dp"
     android:layout_marginHorizontal="2dp"
-    android:background="@android:color/transparent"
-    android:contentDescription="@string/app_name">
+    android:background="@drawable/bg_rich_text_menu_button"
+    android:contentDescription="@string/rich_text_editor_format_bold"
```

Comment:
Better to use `tools:contentDescription`, since this will be changed at runtime.

Reply:
It seems like this is not enough. See https://github.com/vector-im/element-android/pull/7354#discussion_r994694521.